pkg/v1/providers/ytt/vendir/kapp-controller/_ytt_lib/addons/packages/kapp-controller/0.30.0/bundle/config/values.yaml

```diff
@@ -9,7 +9,7 @@ kappController:
   deployment:
     #! The coreDNSIP will be injected into /etc/resolv.conf of kapp-controller pod
     coreDNSIP: null
-    hostNetwork: null
+    hostNetwork: false
```

Comment:
question: I think we are using `hostNetwork:true` with `dnsPolicy:ClusterFirst` for kapp-controller. If assigning `false` is intended, could you please help put some comments ? thanks

Reply:
This is the default value of the package template. The value we use in addon secret is located else where.

Reply:
The value with which we override is located in https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v1/providers/ytt/02_addons/kapp-controller/kapp-controller_addon_data.lib.yaml#L25
We are not changing that.It is `true`